lib/presentation/organisms/ready_for_session_organism.dart

```diff
@@ -25,6 +27,7 @@ class _ReadyForSessionOrganismState extends State<ReadyForSessionOrganism> {
   bool confettiGotPlayed = false;
 
   void onPressed() {
+    widget.onTapReady?.call();
```

Comment:
There is no reason for this method to get called each click.
Move it under the if statement.

Also the name is incorrect, see my other comment.